package.json

```diff
@@ -8,7 +8,8 @@
     "prettier:format": "prettier --write .",
     "prettier:check": "prettier --check .",
     "typecheck": "tsc --build",
-    "validate": "concurrently --kill-others-on-fail -g -p \"[{name}]\" -n \"lint,prettier,typecheck\" \"npm:lint\" \"npm:prettier:check\" \"npm:typecheck\""
+    "validate": "concurrently --kill-others-on-fail -g -p \"[{name}]\" -n \"lint,prettier,typecheck\" \"npm:lint\" \"npm:prettier:check\" \"npm:typecheck\"",
```

Comment:
Please adjust this script to also validate the tests:

```json
"validate": "concurrently --kill-others-on-fail -g -p \"[{name}]\" -n \"lint,prettier,typecheck,test\" \"npm:lint\" \"npm:prettier:check\" \"npm:typecheck\" \"npm:test\"",
```

Reply:
done